src/utils/validate-controls.js

```diff
@@ -460,7 +460,7 @@ const validateSingleSelectControl = (
   templateExceptionMap,
   i18n
 ) => {
-  const { active, available = [], controlId } = control
+  const { active, available = [], controlId, sourcePathMap, ref } = control
   if (!active) {
     addExceptions(
       undefined,
```

Comment:
@chenz4027 I'm still worried about this line actually... not sure why we are trying to add an undefined exception.